app/Services/UserRoleServices.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Services;
+
+use App\Contracts\UserRoleContracts;
+use App\Models\UserRoles;
+use Illuminate\Http\Request;
+
+
+
+class UserRoleServices implements UserRoleContracts
+{
+    public function getRole(array $data): array
+    {
+        $userRole = UserRoles::where('user_email', $data['user_email'])->first();
+        return [
+            'user_role' => $userRole->user_role,
+            'user_email' => $data['user_email']
```

Comment:
add display_name, remember_token, password, to the migration

Reply:
this is resolved!